portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** A comment */
+public final class Comment {
+
+  private final long id;
+  private final String content;
```

Comment:
Does this compile?  I thought that final fields are set when the class is loaded, rather than when the constructor is called.

Reply:
For java, final fields are set when the value is first assigned. This code does compile, but I can see it causing an issue if I wanted to edit a comment in the future

Reply:
I think you can probably just remove the final from the class definition (unless you want to enforce it to not be subclassed) and also remove the final for the string content (which is probably what you'd want to modify in the future, while id can likely remain unmodifiable).

Reply:
Sg